LISA/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ find_package(CompileSettingsDebug CONFIG REQUIRED)
 find_package(CURL REQUIRED)
 find_package(LibArchive REQUIRED)
 find_package(Boost COMPONENTS system filesystem REQUIRED)
+find_package(PkgConfig)
```

Comment:
So maybe create cmake directory and copy FindSqlite.cmake there and here:
find_package(Sqlite REQUIRED) ?

Reply:
done